src/assets/translations/en/main.json

```diff
@@ -2646,13 +2646,15 @@
     "confirmAndBridge": "Confirm & Bridge",
     "myStakedBalance": "My Staked Balance",
     "myStakedBalanceHelper": "The amount of FOX you have staked",
-    "pendingRewardsBalance": "Current Estimated Rewards Balance",
+    "pendingRewardsBalance": "Pending Rewards Balance",
```

Comment:
Was this discussed with @shapeshift/product? We had updated to "Current Estimated" because that is all it is, you may get more or less by epoch close, so to call it "Pending" is a misnomer since the value isn't really truly pending until epoch close.

Reply:
I've taken it from the Figma but it might be outdated, checking with @shapeshift/product which copy is the right one